charmcraft/cmdbase.py

```diff
@@ -21,6 +21,7 @@
 import craft_cli
 
 JSON_FORMAT = "json"
+FORMAT_HELP_STR = "Produce the formatted result in the specified format (currently only 'json')"
```

Comment:
I think this reads better as `Produce the result in the specified format (currently only 'json')`

Reply:
@mr-cal yes, I agree. Updated the string.